mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -203,10 +203,7 @@ protected Boolean doInBackground(String... strings) {
                         Log.d(TAG, "Socket connected");
                         s.close();
                         return true;
-                    } catch (MalformedURLException e) {
-                        Log.e(TAG, e.getMessage());
-                        return false;
-                    } catch (IOException e) {
+                    } catch (Exception e) {
```

Comment:
can this be more clearly caught using java 7 syntax like `catch (MalformedURLException | IOException e)` rather than defaulting to a generic exception?